apps/src/templates/sectionProgress/SummaryViewLegend.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { PropTypes, Component } from 'react';
+import ProgressBox from './ProgressBox';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  legendBox: {
+    borderWidth: '1px',
```

Comment:
nit: `1` instead of `'1px'` for consistency

Reply:
Storybook throws errors without the px (this is the same issue @epeach  and I hit a couple weeks ago).  Not sure if it means we need to update the style guide? 
<img width="1030" alt="screen shot 2018-04-10 at 12 13 21 pm" src="https://user-images.githubusercontent.com/12300669/38578490-e909d8e6-3cb8-11e8-8381-112bd9bf6882.png">

Reply:
I won't like the string `'1'`, but it will like the number `1`